frame.test.js

```diff
@@ -0,0 +1,47 @@
+const Frame = require('./frame');
+const Roll = require('./roll');
+
```

Comment:
Using doubles in your tests would ensure your classes aren't dependent on each other to work, the way you've set up your constructors should mean you could use dependency injection to do this?